scrapy_poet/page_input_providers.py

```diff
@@ -8,12 +8,25 @@
 import abc
 import typing
 
-from scrapy.http.response import Response
+from scrapy import Spider
+from scrapy.http import Request, Response
+from scrapy.crawler import Crawler
+from scrapy.statscollectors import StatsCollector
+from scrapy.settings import Settings
 from web_poet.page_inputs import ResponseData
 
 # FIXME: refactor _providers / provides / register,  make a nicer API
 providers = {}
 
+SCRAPY_PROVIDED_CLASSES = {
```

Comment:
What do you think about making this constant private (_SCRAPY_PROVIDED_CLASSES)? I'm not sure it should be a part of the public API, or do you see a use case for that?

Reply:
Another idea: move it to utils. Currently this module contains provider themselves, while in utils there are functions to create providers and instances.

Reply:
Done.

- moved to util.py
- turned into a private constant